doc/2/guides/write-plugins/old-guides/index.md

```diff
@@ -2,6 +2,12 @@
 code: false
 type: branch
 order: 400
-title: Old Guides
-description: Kuzzle Old Plugins guides
+title: Old Guides | Write plugins | Guide 
+meta:
+  - name: description
+    content:  Kuzzle Old Plugins guides
+  - name: keywords
+    content: Kuzzle, Documentation, kuzzle write pluggins, old guides
 ---
+
+<Redirect to="hooks" />
```

Comment:
Is this the write redirect ?

Reply:
Yes, I doubled checked in the code and on the pre prod and it's redirecting on hooks 
https://next-docs.kuzzle.io/core/2/guides/write-plugins/old-guides